examples/gcoap/client.c

```diff
@@ -180,17 +167,37 @@ static size_t _send(uint8_t *buf, size_t len, char *addr_str)
 
 static int _print_usage(char **argv)
 {
-    printf("usage: %s <get|post|put|ping|proxy|info>\n", argv[0]);
+    printf("usage: %s <get [-o|-d]|post|put|ping|proxy|info>\n", argv[0]);
```

Comment:
Nit: Could a CLI like `coap observe ::1` be more intuitive than you proposed `coap get -o ::1`?